src/vs/workbench/browser/parts/editor/editorActions.ts

```diff
@@ -1087,6 +1112,36 @@ export class MaximizeGroupAction extends Action2 {
 	}
 }
 
+export class UnmaximizeEditorGroupAction extends Action2 {
+
+	constructor() {
+		super({
+			id: UNMAXIMIZE_EDITOR_GROUP,
+			title: { value: localize('UnmaximizeEditorGroup', "Unmaximize Editor Group"), original: 'Unmaximize Editor Group' },
+			f1: true,
+			category: Categories.View,
+			precondition: MaximizedEditorGroupContext,
+			keybinding: {
+				weight: KeybindingWeight.WorkbenchContrib,
+				primary: KeyChord(KeyMod.CtrlCmd | KeyCode.KeyK, KeyMod.CtrlCmd | KeyCode.KeyM),
+			},
+			menu: {
+				id: MenuId.EditorTitle,
+				order: -10000, // towards the front
+				group: 'navigation',
+				when: MaximizedEditorGroupContext
+			},
+			icon: Codicon.screenFull,
```

Comment:
Is this the final icon or are we thinking to ask for a new one?

Reply:
What do youn think? I think it's oke

Reply:
Yeah, maybe @hbons or @daviddossett have thoughts about this icon to be used in this context